esmvaltool/preprocessor/derived_variables/clhmtisccp.py

```diff
@@ -0,0 +1,64 @@
+"""Derivation of variable `clhmtisccp`."""
+
+
+import iris
+from iris import Constraint
+
+from ._derived_variable import DerivedVariable
+
+
+class clhmtisccp(DerivedVariable):  # noqa
```

Comment:
Please use class names that conform with PEP8.

Reply:
> This PR adds 500 lines of code, almost doubling the size of the module, without adding new functionality. Can you try to reduce the amount of boilerplate/duplicated code and documentation so we get back a to a number of lines of code similar to what we had before?

I hardly added any code (I did not change the actual derive functions), so those 500 lines more are mostly docstrings.

> Wouldn't it make more sense to organize the derived variables in the same way as the cmor tables, i.e. in a cmor_table/mip structure?

> One file per derived variable seems very fine grained, some derivation functions are really are just one cube minus another one, that's just four lines of code.

But which variables should be combined in one file? All of them? I thought we wanted to separate them?

Reply:
no, we need to name the class the same way as the variable - this is what is in `fixes` as well

Reply:
I can rename the base class to `DerivedVariableBase` and every child class `DerivedVariable`; the name of variable is already the name of the file/module. Is this an option?